Add aeson-yak

tejon opened the pull request:
Add package: aeson-yak (tanuki@gmail.com)

lwm commented:
Ah, that bounds issue is unrelated, will fix and re-run this build.

lwm commented:
Oh damn, actually, @tejon, you've got bounds issues with `aeson`, can you fix those up?

```
aeson-1.2.3.0 (Adam Bergmark <adam@bergmark.nl> @bergmark) is out of bounds for:                                                                                                               
- [ ] aeson-yak-0.1.1.1 (>=0.4 && < 0.12). Theodore Lief Gannon <tanuki@gmail.com> @tejon. @tejon. Used by: library   
```

I've blocked `aeson-yak` since placing an upper bound on `aeson` is not fun.

tejon commented:
Whoops, you're still seeing 0.1.1.1 -- before submitting, I cleaned up the bounds in 0.1.1.2. Then discovered haddocks were broken, and fixed them in 0.1.1.3. :P I waited to submit until stack could find 0.1.1.3 via extra-deps, but I guess not long enough for other mirrors.

lwm commented:
Unblocked! Thanks 👍
